docker/Dockerfile.server

```diff
@@ -0,0 +1,13 @@
+FROM bcgovimages/von-image:node-1.12-6
+
+ENV LOG_LEVEL ${LOG_LEVEL:-info}
+ENV RUST_LOG ${RUST_LOG:-warning}
+
+ADD server/requirements.txt server/
+RUN pip3 install -U pip && \
+    pip install --no-cache-dir -r server/requirements.txt
+
+ADD --chown=indy:indy config ./config
+ADD --chown=indy:indy server ./server
+
+CMD python3 -m server.server
```

Comment:
The change to the startup `cmd` complicates the genesis file management a bit.

Reply:
Removed as we reverted base images separation.